core/src/main/java/apoc/temporal/TemporalProcedures.java

```diff
@@ -80,6 +126,33 @@ public String format(
      * @return
      */
     @UserFunction("apoc.temporal.formatDuration")
+    @QueryLanguageScope(scope = {QueryLanguage.CYPHER_5})
+    @Description("Formats the given duration into the given time format.")
+    public String formatDurationCypher5(
+            @Name(value = "input", description = "The duration value to be formatted into a string.") Object input,
+            @Name(value = "format", description = "The format to return the duration in.") String format) {
+        DurationValue duration = ((DurationValue) input);
+
+        try {
+            String pattern = getOrCreateDurationPattern(format);
+            return getDurationFormat(duration, pattern);
+        } catch (Exception e) {
+            throw new RuntimeException("Available formats are:\n" + String.join("\n", getTypes())
+                    + "\nSee also: https://www.elastic.co/guide/en/elasticsearch/reference/5.5/mapping-date-format.html#built-in-date-formats "
+                    + "and https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html");
+        }
+    }
+
+    /**
+     * Convert a Duration into a LocalTime and format the value as a String
+     *
+     * @param input
+     * @param format
+     * @return
+     */
```

Comment:
Probably don't need this comment, or at least not all the param parts as they are empty